app/.eslintrc

```diff
@@ -0,0 +1,58 @@
+{
+  "env": {
+    "amd": true,
+    "es6": true,
+    "jasmine": true,
+    "jquery": true,
+    "node": true,
+    "mocha": true,
+    "browser": true,
+    "builtin": true
+  },
```

Comment:
Setting Node: false actually throws 21 errors of "no-console" in the app/ directory.. What should I do? 
@pdehaan